app/assets/javascripts/discourse/app/components/dialog-messages/confirm-session.gjs

```diff
@@ -119,12 +148,20 @@ export default class ConfirmSession extends Component {
               @label="user.password.confirm"
             />
           </div>
+          <div class="confirm-session__reset">
+            <a href {{on "click" this.sendPasswordResetEmail}}>
```

Comment:
Is there a reason we are using a `<a>` tag here instead of a button which is the more appropriate element to use?

Reply:
Yes, this is a small element, just below the password input, it looks very much like a link and not a button. See updated screenshots in OP.

Reply:
If it isn’t use for navigation then we should use a button, I believe we have an existing CSS that makes the button look like a link.

Reply:
Yeah, fair enough, switched to a button.